Armadillo/src/main/java/com/scribd/armadillo/playback/mediasource/HlsMediaSourceGenerator.kt

```diff
@@ -28,6 +29,11 @@ internal class HlsMediaSourceGenerator @Inject constructor(
                 return DownloadHelper.createMediaSource(it.request, dataSourceFactory)
             }
         }
+
+        if (request.drmInfo != null) {
+            Log.e(MediaSourceGenerator.TAG, "HLS does not currently support DRM")
```

Comment:
@kschults Might be good to throw here cause we wouldn't want someone to think they are getting DRM protections and then still play the content. wyt?

Reply:
Damn, dude force ghosted in to call out a customer concern

:,) 🙏